failing CI due to small diff in matplotlib results

* failing CI due to small diff in matplotlib results
* https://github.com/uscuni/simplification/actions/runs/18289716351/job/52073664489


```
=========================== short test summary info ============================
FAILED core/tests/test_viz_context.py::test_viz_context_param_plot_video[png] - matplotlib.testing.exceptions.ImageComparisonFailure: images not close (RMS 3.560):
	result_images/test_viz_context/test_viz_context_param_plot_video.png
	result_images/test_viz_context/test_viz_context_param_plot_video-expected.png
	result_images/test_viz_context/test_viz_context_param_plot_video-failed-diff.png
FAILED core/tests/test_viz_h3_hex.py::test_viz_h3_hex_plot_aoi[png] - matplotlib.testing.exceptions.ImageComparisonFailure: images not close (RMS 2.570):
	result_images/test_viz_h3_hex/test_viz_h3_hex_plot_aoi.png
	result_images/test_viz_h3_hex/test_viz_h3_hex_plot_aoi-expected.png
	result_images/test_viz_h3_hex/test_viz_h3_hex_plot_aoi-failed-diff.png
FAILED core/tests/test_viz_h3_hex.py::TestVizH3HexPlotCell::test_loc_id[png] - matplotlib.testing.exceptions.ImageComparisonFailure: images not close (RMS 3.166):
	result_images/test_viz_h3_hex/test_viz_h3_hex_plot_cell_loc_id.png
	result_images/test_viz_h3_hex/test_viz_h3_hex_plot_cell_loc_id-expected.png
	result_images/test_viz_h3_hex/test_viz_h3_hex_plot_cell_loc_id-failed-diff.png
```

--- a/core/tests/test_viz_context.py
+++ b/core/tests/test_viz_context.py
@@ -39,7 +39,8 @@ def test_viz_context_location():
 
 
 # RMS (12.174 // 255) for Ubuntu DEV --> MPL via pypi (very small diff)
-tol = 12.2 if pytest.ENV_TYPE == "dev" else 3
+# -- more diff -- simplification#285
+tol = 12.2 if pytest.ENV_TYPE == "dev" else 4
 
 
 @image_comparison(

--- a/core/tests/test_viz_h3_hex.py
+++ b/core/tests/test_viz_h3_hex.py
@@ -4,10 +4,12 @@ from matplotlib.testing.decorators import image_comparison
 import core
 
 
+# (RMS 6.914)
+# -- more diff -- simplification#285
 @image_comparison(
     baseline_images=["test_viz_h3_hex_plot_aoi.png"],
     style="mpl20",
-    tol=7 if pytest.ENV_TYPE == "dev" else 2,  # (RMS 6.914)
+    tol=7 if pytest.ENV_TYPE == "dev" else 3,
 )
 def test_viz_h3_hex_plot_aoi(grid_8_auckland):
     core.viz.h3_hex.plot_aoi(grid_8_auckland, "Auckland", 8)
@@ -38,7 +40,9 @@ def test_viz_h3_hex_plot_analysis(manual_auckland, parenx_auckland, grid_7_auckl
     core.viz.h3_hex.plot_analysis(grid, f"{var}_ratio", info=f"({_p}/{_m})")
 
 
-tol = 5 if pytest.ENV_TYPE == "dev" else 3  # (RMS 4.030)
+# (RMS 4.030)
+# -- more diff -- simplification#285
+tol = 5 if pytest.ENV_TYPE == "dev" else 4
 
 
 class TestVizH3HexPlotCell:

